### DISCUSSION TITLE
Add demonstration for QSVM

### DISCUSSION THREAD

**Discussion by ashmitjsg** [devilkiller-ag/UnravelQuantum#4] [Ideas]
Add demonstration for QSVM

This page should have:

- Demonstration (Users should be able to tweak options.Show the output circuit, measurement results, histogram to the user.)
- Detailed explanation including mathematics and examples (wherever required) divided into multiple sections
- Generalised and modular code written (See the code given in Bernstein Vazirani and Deutsch Josza Algo Pages for reference)
- Links to external sources and further study material
- About the author (small description and socials) 

Example:
Implementation of [Deutsch Josza Algo ](https://github.com/devilkiller-ag/UnravelQuantum/blob/main/pages/1_%E2%9A%94%EF%B8%8F_Deustch_Josza_Algorithm.py). Here is the page for the [Deutsch Josza Algo Demonstration](https://unravelquantum.streamlit.app/Deustch_Josza_Algorithm) created from this code.

**Comment by karthiganesh** [comment]
Hi ,

I propose an idea to explain Quantum Support Vector Machine with IRIS Dataset.

**Reply by ashmitjsg** [reply]
Sure @karthiganesh. I will create an issue for this and assign it to you.

**Reply by ashmitjsg** [reply]
@karthiganesh I have opened an issue #5 regarding it. I recommend you to create a draft PR for your work for monitoring and discussion purposes.
